src/Serializer/AbstractItemNormalizer.php

```diff
@@ -89,12 +92,25 @@ public function supportsNormalization(mixed $data, string $format = null, array
 
     /**
      * {@inheritdoc}
+     * @deprecated since Symfony 6.3, use "getSupportedTypes(?string $format)" instead
      */
     public function hasCacheableSupportsMethod(): bool
     {
+        trigger_deprecation('symfony/serializer', '6.3', 'The "%s()" method is deprecated, use "getSupportedTypes()" instead.', __METHOD__);
+
         return true;
     }
 
+    public function getSupportedTypes(?string $format)
+    {
+        return [
```

Comment:
I'm really not sure about this modification.

I've tried to understand the purpose of the getSupportedTypes method and adapted it to our class.

I'm waiting for your feedback to continue on the subject.